start-gpu.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+BASE_DIR="/home/dwemer"
+REPO_URL="https://github.com/Dwemer-Dynamics/chatterbox"
```

Shellcheck (0.11.0)

[warning] REPO_URL appears unused. Verify use (or export if used externally).

(SC2034)